### DISCUSSION TITLE
Server related question

### DISCUSSION THREAD

**Discussion by milly531** [RealMCoded/LunarRec#11] [Q&A]
I might sound dumb but how do others connect to the server? when i look at the files it says from what i saw, it is hosted on localhost and other people cant connect to localhost, unless it is like rebornrec and you start the server and play with other people on localhost, or is the server on the website you change from "Example.com" in config.json.

**Comment by milly531** [comment]
Also, where would one source the builds to support the LunarRec Server?(On localhost) or do you have to make em your self

**Comment by RealMCoded** [comment]
> I might sound dumb but how do others connect to the server? when i look at the files it says from what i saw, it is hosted on localhost and other people cant connect to localhost

The only mention of localhost there is happens inside of the template config. that mention of localhost can be changed to whatever domain you are hosting it on (or IP if you dont have a domain). The only time this is used however is when the game requests the name server. For example, if your ip was set to `http://192.168.14.46:8080`  in the config (i know this is a local ip), the server would send this as the name server:
```
{NOTE: "LunarRec Name Server. If IPs are wrong check your config.", API:`http://192.168.14.46:8080`, Notifications:`http://192.168.14.46:8080`, Images:`http://192.168.14.46:8080/img`}
```

The server only hosts on the the given port in the config. so if it was port 8080, you would need to port forward port 8080. i hope this clears it up but if you want more details on some parts, let me know.

> Also, where would one source the builds to support the LunarRec Server?(On localhost) or do you have to make em your self

You must make them yourself. I don't provide builds because of Rec Room's Copyright over builds.

**Reply by milly531** [reply]
Im not really tech savvy but, does the domain work on anykind of domain?(other kinds of domains work on rec room bcuz epicquests server used to be hosted on EpicQuest.live) but does it work on domains like .xyz or does it work with all domains

**Comment by RealMCoded** [comment]
Yes, it should work on all types of domains. I used that one as an example.  On Aug 30, 2024 12:21 AM, RitzDx ***@***.***> wrote:
Im not really tech savvy but, does the domain work on anykind of domain?(other kinds of domains work on rec room bcuz epicquests server used to be hosted on EpicQuest.live) but does it work on domains like .xyz or does it work with all domains
